### DISCUSSION TITLE
What is this url http://featherfiles.aviary.com in admin/filemanager/ajax_calls.php

### DISCUSSION THREAD

**Discussion by apohutenumera** [PrestaShop/PrestaShop#36561] [Q&A]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Hi,
i don't understand why there is this url http://featherfiles.aviary.com in admin/filemanager/ajax_calls.php in 1.7.8.11 zip file ?

### Steps to reproduce

1. Download 1.7.8.11 zip release file
2. go to admin/filemanager and open ajax_calls.php
3.  check line 39. This url appears.

### PrestaShop version(s) where the bug happened

1.7.8.11

**Comment by matks** [comment]
Hello @apohutenumera , the filemanager JavaScript code is very old (around 10 years), so I'm afraid the people who knew are not there to answer today.

If I google it, I can see Feather Files used to be a Image Bank so this line of code was probably written to handle files coming from that Image Bank. Today the website http://featherfiles.aviary.com/ so this code could go away.
